frontend/persist.lua

```diff
@@ -0,0 +1,131 @@
+local bitser = require("ffi/bitser")
+local dump = require("dump")
+local lfs = require("libs/libkoreader-lfs")
+
+local codecs = {
+    -- bitser: binary form, fast encode/decode, low size. Not human readable.
+    bitser = {
+
+        serialize = function(t, file)
```

Comment:
(I'm not fond of these blank lines at start and end, but I'm fine when in the middle :) but that's just me.)